### DISCUSSION TITLE
assign virtual machine to device

### DISCUSSION THREAD

**Discussion by nokiaIL** [netbox-community/netbox#17027] [Q&A]
hello, 
I have a lab with physical servers that run ESXI.
each server is standalone, with no Vcenter, etc.
how can I assign a virtual machine directly to a physical server without creating a cluster per server? 

thanks,

**Comment by candlerb** [comment]
> how can I assign a virtual machine directly to a physical server without creating a cluster per server?

Modern versions of Netbox let you put all your VM hosts into one big cluster, and then assign the VMs to a specific device:

![image](https://github.com/user-attachments/assets/60f54f23-800c-409d-8c63-a684b45f610d)

**Reply by nokiaIL** [reply]
thank you, a great solution for me.
